source/includes/quick-start/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	uri         = os.Getenv("MONGODB_URI")
+	ctx, cancel = context.WithTimeout(context.Background(), 20*time.Second)
+)
+
+func main() {
+	defer cancel()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		log.Panic(err)
+	}
+
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			log.Panic(err)
+		}
+	}()
+
+	db := client.Database("sample_mflix")
```

Comment:
nit: you can call `coll := client.Database("sample_mflix").Collection("movies")` as one line